### ISSUE TITLE
Timeline about the BabylonJS V5 integration with the BabylonJS Editor 

### ISSUE THREAD

**Issue by eherozhao** [BabylonJS/Editor#377]
Hello @julien-moreau , we hit some animation issues when importing glTF asset into Editor. Basically we just want to import the static mesh but didn't want the animation inside the glTF to play automatically. Refer to this forum question: https://forum.babylonjs.com/t/how-to-import-a-gltf-without-its-animation/30263 for more contexts. 

The solutions now couldn't solve our issue perfectly. So I would like to see whether the BabylonJS v5 integration with BabylonJS Editor will solve this issue. If it is, do you have an approximate timeline? Thank you.

**Comment by julien-moreau** [comment]
Hi @eherozhao !
I plan to release the editor with Babylon.JS 5 this weekend

In the code right now I see
```typescript
const result = await SceneLoader.ImportMeshAsync("", join(dirname(this.props.absolutePath), "/"), basename(this.props.absolutePath), scene);
scene.stopAllAnimations();
```

That means all animations are stopped automatically. Is that possible you send me the sample model you use so I can try to fix before the release?

Thanks a lot

**Comment by eherozhao** [comment]
Hi @julien-moreau !

Thank you for the reply. You can get the asset model here: https://github.com/aws-samples/amazon-sumerian-hosts/tree/mainline2.0/packages/demos-babylon/src/character-assets. They are all open source assets. If you need more contexts, please let me know.

**Comment by julien-moreau** [comment]
Thanks for sharing @eherozhao !
I'm having issues with the latest 5.7.0 version of Babylon.JS, I'm looking for the problem but the release may not be this week-end.

**Comment by julien-moreau** [comment]
@eherozhao hi !
The Editor using BabylonJS v5 has just been released (v4.3.0)
I'm fixing (it not already fixed) your issue and will deploy this fix in 4.3.1

**Comment by eherozhao** [comment]
@julien-moreau Thank you so much!! I will test it on my side once the 4.3.1 is posted.
